docs/components/material.md

```diff
@@ -516,26 +523,248 @@ void main() {
 }
 ```
 
+To use these vertex and fragment shaders, 
+after reading them into strings `vertexShader` and `fragmentShader`,
+we simply register our custom shader with A-Frame:
+
+```js
+// shader-grid-glitch.js
+
+AFRAME.registerShader('grid-glitch', {
+  schema: {
+    color: {type: 'color', is: 'uniform'},
+    timeMsec: {type: 'time', is: 'uniform'}
+  },
+
+  vertexShader: vertexShader,
+  fragmentShader: fragmentShader
+});
+```
+
 Finally, here is the HTML markup to put it all together:
 
 ```html
 <!-- index.html -->
 
 <a-scene>
-  <a-sphere material-grid-glitch="color: blue;"
+  <a-sphere material="shader:grid-glitch; color: blue;"
             radius="0.5"
             position="0 1.5 -2">
   </a-sphere>
 </a-scene>
 ```
 
-* [Live demo](https://aframe-simple-shader.glitch.me/)
-* [Remix this on Glitch](https://glitch.com/edit/#!/aframe-simple-shader)
+* [Live demo](https://aframe-simpler-shader.glitch.me/)
```

Comment:
All the Glitch demos can be listed in the `examples` array in the header:

```
examples:
  - title: Grid Glitch Shader
    url: https://glitch.com/edit/#!/aframe-simpler-shader?path=client/shader-grid-glitch.js:1:0
```

For ease of copy/pasting, I'd remove the GLSLify build step and use [multiline strings](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#Multi-line_strings).

Reply:
It was that way originally (using GLSL-ify) so I left it that way; you can see that the new one I added as walkthrough does, in fact, use multi-line strings.  Are you asking to make changes to the existing example instances that will retroactively alter documentation for prior versions?

Reply:
You can remix the Glitch and not alter the old one.

Reply:
FYI - current examples header looks pretty unattractive, and no other good examples to follow?

Reply:
It's a data structure, it gets rendered underneath the table of contents as a list. I think there are two Glitches you listed, can add more as we go.

Reply:
Just checked, there are more than two. Good list!

Reply:
updated the Glitches I made, just the page and script, no more extraneous server and files